doc/api/stream.md

```diff
@@ -1197,7 +1197,10 @@ print(fs.createReadStream('file')).catch(console.log);
 
 If the loop terminates with a `break` or a `throw`, the stream will be
 destroyed. In other terms, iterating over a stream will consume the stream
-fully.
+fully. 
+Note that by default, async reading of the contents of the file will be
```

Comment:
Nit: it should be obvious that this is a `async` operation, so I would not mention that again.

Reply:
I still find this a bit confusing. For example, "of the file"...there's no guarantee the stream is a file, or am I wrong about that?

How about this?

> By default, the stream will be read in chunks of size equal to the `highWaterMark` value which defaults to...

Reply:
(Maybe omit the mention of the default value from `fs.createReadStream()` since the default value is 16kb for some other ways of creating streams.)

Reply:
Is "by default" even necessary in the sentence? Is there an option to change it? If so, maybe it should be mentioned? "If `fooOption` is not set, then by default, the stream will be..."

Reply:
> For example, "of the file"...there's no guarantee the stream is a file, or am I wrong about that?

Its just that I tested with `fs.createReadStream()` and the same example in docs but I guess this behaviour is the same in other cases

After thinking this though I think you are right in all of your suggestions. But note became too general so I would also add a test-case example:
> The stream will be read in chunks of size equal to the `highWaterMark` option.
So in code example above it will only be one iteration if `'file'` has less
then 64kb of data. Unless custom `highWaterMark` value is specified in
[`fs.createReadStream()`][] constructor.

Looks good?
If its too much hassle I can just left general part:
> The stream will be read in chunks of size equal to the `highWaterMark` option.

Reply:
Maybe this?

> The stream will be read in chunks of size equal to the `highWaterMark` option.
> In the code example above, data will be in a single chunk if 'file' has less
> then 64kb of data because no `highWaterMark` option is provided to
> [fs.createReadStream()][].

Reply:
Seems good 👍 
thank you, updated in https://github.com/nodejs/node/pull/19331/commits/3aa49be2f335d295c898f20086b53669d72f8b24
write good docs is hard 😄

Reply:
Thanks for your patience. The PR-review process for docs can seem tedious and endless sometimes.